### DISCUSSION TITLE
Minimum configuration

### DISCUSSION THREAD

**Discussion by cleuson-alves** [apiman/apiman#2582] [General]
Hello, I would like to know the minimum requirements for creating a Linux virtual machine running the apiman server, how much disk space, hard disk and RAM memory. Thanks

**Comment by msavy** [comment]
Hi, Cleuson!

Unfortunately, that's very much an "it depends" question. For example, are you running Elasticsearch for your metrics?

When you say the Apiman server, do you mean just the Apiman Manager or also a gateway?

Which gateway implementation are you planning on using? Vert.x, WildFly, Tomcat?

How much traffic are you expecting to run through the gateway (e.g. transactions per second)?

Let me know your proposed architecture/setup.
